Documentation/configuration/vlan-802.1q.rst

```diff
@@ -14,7 +14,7 @@ Cilium enables firewalling on native devices in use and will filter all unknown
 will always be passed through their main device with associated tag (e.g. VLAN device is ``eth0.4000`` and its main interface is ``eth0``).
 By default, Cilium will allow all tags from the native devices (i.e. if ``eth0.4000`` is controlled by Cilium and has
 an eBPF program attached, then VLAN tag ``4000`` will be allowed on device ``eth0``). Additional VLAN tags may be allowed
-with the cilium-agent flag ``--vlan-bpf-bypass=4001,4002`` (or Helm variable ``--set bpf.vlan-bpf-bypass=4001,4002``).
+with the cilium-agent flag ``--vlan-bpf-bypass=4001,4002`` (or Helm variable ``--set bpf.vlanBypass="4001 4002"``).
```

Comment:
I don't think the syntax for helm is no different with string value :thinking: , while this field is array type.